components/resources/ResourceOccurrences.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Loading from "../Loading";
+import ResourceOccurrenceCard from "./ResourceOccurrenceCard";
+import React from "react";
+import PropTypes from "prop-types";
+import { useFetch } from "hooks/useFetch";
+
+const ResourceOccurrences = (props) => {
+  const { resourceUri } = props;
+
+  const { data, loading } = useFetch(
+    resourceUri
+      ? `/api/occurrences?resourceUri=${encodeURIComponent(resourceUri)}`
```

Comment:
Could be a future improvement, but it would be nice if `useFetch` took an object of query params and it could handle calling `encodeURIComponent` or [new'ing up `URLSearchParams`](https://stackoverflow.com/a/58437909).

Reply:
Good idea - I will do this in my next set of changes.